Podfile

```diff
@@ -39,10 +39,10 @@ def aztec
 end
 
 def wordpress_ui
-    pod 'WordPressUI', '~> 1.12.1'
+    #pod 'WordPressUI', '~> 1.12.1'
     #pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS', :tag => ''
     #pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS', :branch => ''
-    #pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS', :commit => ''
+    pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS', :commit => 'fd409ce81111e9870022a18d8c384b51ebe58798'
```

Comment:
FYI: I took the liberty to add the `[Status] Blocked` label to be sure we don't accidentally forget to make a new WPUI release and update this reference here before merging

Reply:
There's a "DO NOT MERGE" label that seems to raise a peril alert.  I'd guess this is more appropriate for the case?

Reply:
Yeah, to be honest those two labels are pretty close to one another semantically, I think they are sometimes used interchangeably. The fact that the "Do Not Merge" label raises a peril alert is a good point and argument for it tho, indeed 👍 

Though, that being said, I also forgot we also [already have a Peril rule which detected when we have a `pod` pointing to a commit](https://github.com/Automattic/peril-settings/blob/master/org/pr/ios-macos.ts#L17-L22) – which was actually triggered before I added my label or yours, according to my email notification history – so I think if we don't set any of those labels (which seems to take precedence over that other rule), Peril will detect it and still ensure we don't accidentally merge a pod pointing to a commit anyway 😅 

Anyway, as long as we make it clear to anyone reviewing the PR that it should not be merged until we update this, that's what's important 🙂